guide/daily-spending-wallet/security.md

```diff
@@ -362,6 +371,18 @@ By including UX patterns for hiding information pattern wallets, we give users a
 
 ---
 
+## Preventing theft when offline
+
+Mobile wallets may frequently be offline, either due to a poor connection or because the application is not running. To improve battery live, mobile operating systems tend to prevent applications the user is not actively using from running code in the background. This can prevent mobile wallets from being reliable monitors of its payment channels. Malicious actors who share channels with the user may take advantage of these moments to try to steal funds by closing channels with incorrect data.
+
+A simple technique to reduce this risk is to notify the user when the application has been offline for an extended period of time. [Forced channel closes](https://wiki.ion.radar.tech/tech/channels/channel-closing#force-close) have an agreed upon delay built-in. Notifications should be sent early enough so that users have time to appropriately respond.
+
+[Watchtowers](https://wiki.ion.radar.tech/tech/research/watchtowers) address this problem by continuously monitoring wallets and securely acting on the owners behalf when a problematic situation occurs. Watchtowers services should be provided by a different party than the wallet and payment channels, to reduce the risk of collusion.
+
+{% include image-gallery.html pages = page.images_watchtower %}
```

Comment:
The contrast of the grey used on the iPhone frame and the background is not very good. It does not meet the [WCAG contrast guidelines](https://usecontrast.com/guide)  - we should just remove that grey background ;) https://github.com/BitcoinDesign/Guide/issues/674. There is is a lot frames with this issue fyi - can make another issue for this.

Reply:
WCAG contrast guidelines are for text, not for subtle background colors. This is a separate issue.

Also I think WCAG 2.0 needs to be interpreted as a guideline. My main problem with them is that text sizes 18 and up is treated the same. But, for example, 64pt text is so much larger that lower contrast still works. If we want a very-high-contrast version we should implement [prefers-contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast), a CSS media query that allows users/browsers to instruct websites to beef up the contrast. It has [80% support](https://caniuse.com/?search=prefers-contrast).